plugins/modules/ocapi_command.py

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022 Western Digital Corporation
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: ocapi_command
+version_added: 6.3.0
+short_description: Manages Out-Of-Band controllers using Open Composable API (OCAPI)
+description:
+  - Builds OCAPI URIs locally and sends them to remote OOB controllers to
+    perform an action.
+  - Manages OOB controller such as Indicator LED, Reboot, Power Mode, Firmware Update.
+options:
+  category:
+    required: true
+    description:
+      - Category to execute on OOB controller.
+    type: str
+  command:
+    required: true
+    description:
+      - Command to execute on OOB controller.
+    type: str
+  baseuri:
+    required: true
+    description:
+      - Base URI of OOB controller.
+    type: str
+  proxy_slot_number:
+    description: For proxied inband requests, the slot number of the IOM.  Only applies if I(baseuri) is a proxy server.
+    type: int
+  update_image_path:
+    required: false
+    description:
+      - For FWUpload, the path on the local filesystem of the firmware update image.
+    type: str
+  job_name:
+    required: false
+    description:
+      - For DeleteJob command, the name of the job to delete.
+    type: str
+  username:
+    required: true
+    description:
+      - Username for authenticating to OOB controller.
+    type: str
+  password:
+    required: true
+    description:
+      - Password for authenticating to OOB controller.
+    type: str
+  timeout:
+    description:
+      - Timeout in seconds for URL requests to OOB controller.
+    default: 10
+    type: int
+
+author: "Mike Moerk (@mikemoerk)"
+'''
+
+EXAMPLES = '''
+  - name: Set the power state to low
+    community.general.ocapi_command:
+      category: Chassis
+      command: PowerModeLow
+      baseuri: "{{ baseuri }}"
+      username: "{{ username }}"
+      password: "{{ password }}"
+  - name: Set the power state to normal
```

Comment:
Adding newlines between the tasks makes the examples a bit easier to read.
```suggestion

  - name: Set the power state to normal
```